crates/core/component/ibc/src/component/rpc.rs

```diff
@@ -0,0 +1,21 @@
+use penumbra_storage::Storage;
+use tonic::transport::server::Routes;
+
+mod client_query;
+mod connection_query;
+mod consensus_query;
+
+// TODO: hide and replace with a routes() constructor that
+// bundles up all the internal services
+#[derive(Clone)]
+pub struct IbcQuery(penumbra_storage::Storage);
+
+impl IbcQuery {
+    pub fn new(storage: penumbra_storage::Storage) -> Self {
+        Self(storage)
+    }
+}
+
+pub fn routes(_storage: Storage) -> Routes {
+    unimplemented!("functionality we need is only in tonic 0.10")
+}
```

Comment:
The hope here was to be able to expose just an `rpc::routes()` method that would return a partial bundle of `Service`s. This would avoid the code in `pd/main.rs` from having to recite them all, but that API was only added in Tonic 0.10, so it needs #3078 for that approach to work.